Podcast/ExternalProfileViewController.swift

```diff
@@ -93,47 +104,96 @@ class ExternalProfileViewController: UIViewController, UITableViewDataSource, UI
-        user.numberOfFollowing = 100
-        user.numberOfFollowers = 4
+        let userRequest = UserByIDEndpointRequest(userID: id)
+        userRequest.success = { (endpointRequest: EndpointRequest) in
+            DispatchQueue.main.async {
+                if let results = endpointRequest.processedResponseValue as? User {
+                    self.user = results
+                    self.createSubviews()
+                    self.updateViewWithUser(results)
+                    UIApplication.shared.statusBarStyle = .lightContent
+                }
+            }
+        }
         
-        let s = Series()
-        s.title = "Design Details"
-        s.numberOfSubscribers = 832567
-        user.subscriptions = Array(repeating: s, count: 7)
+        System.endpointRequestQueue.addOperation(userRequest)
         
-        let episode = Episode()
-        episode.title = "Puppies Galore"
-        episode.seriesID = ""
-        episode.dateCreated = Date()
-        episode.descriptionText = "We talk lots about dogs and puppies and how cute they are and the different colors they come in and how fun they are."
-        episode.tags = [Tag(name:"Design"), Tag(name:"Learning"), Tag(name: "User Experience"), Tag(name:"Technology"), Tag(name:"Innovation"), Tag(name:"Dogs")]
-        user.favoriteEpisodes = Array(repeating: episode, count: 5)
+        // Now request user subscriptions and favorites
+//        let favoritesRequest = GetUserFavoritesEndpointRequest(userID: id)
+//        favoritesRequest.success = { (favoritesEndpointRequest: EndpointRequest) in
+//            DispatchQueue.main.async {
+//                guard let results = favoritesEndpointRequest.processedResponseValue as? [Episode] else { return }
+//                self.favorites = results
+//                self.profileTableView.reloadData()
+//            }
+//        }
+//        System.endpointRequestQueue.addOperation(favoritesRequest) // UNCOMMENT WHEN FAVORITES ARE DONE
         
-        updateViewWithUser(user)
+        let subscriptionsRequest = FetchUserSubscriptionsEndpointRequest(userID: id)
+        subscriptionsRequest.success = { (subscriptionsEndpointRequest: EndpointRequest) in
+            DispatchQueue.main.async {
+                guard let results = subscriptionsEndpointRequest.processedResponseValue as? [SubscriptionSeries] else { return }
+                self.subscriptions = results
+                self.profileTableView.reloadData()
+            }
+        }
+        System.endpointRequestQueue.addOperation(subscriptionsRequest)
     }
     
     func updateViewWithUser(_ user: User) {
         self.user = user
         // Update views
         profileHeaderView.setUser(user)
         miniHeader.setUser(user)
-        profileTableView.reloadData()
     }
     
     func didPressBackButton() {
-        let _ = navigationController?.popViewController(animated: true)
+        if let navController = navigationController {
```

Comment:
let _ = navigationController?.popViewController(animated: true) should not give a warning